tox.ini

```diff
@@ -35,7 +35,9 @@ deps =
 commands = flake8 flake8_import_order/
 
 [testenv:setuppy]
-deps = docutils
+deps =
+    docutils
+    Pygments
```

Comment:
@sigmavirus24 PyPI supports `.. code-block::` directive and it's even colored. 😄 GitHub also supports it and it's colored too.